### ISSUE TITLE
I'm having trouble with summoning rituals, NBT, and apoli powers

### ISSUE THREAD

**Issue by 800020h** [AlmostReliable/summoningrituals-forge#17]
### Version

1.19.2-2.0.4

### Describe the bug!

some NBT refuses to work

here is the last thing I tried
```js
ServerEvents.recipes(event => {

  event.recipes.summoningrituals.altar("minecraft:bone")
    .mobOutput(
      SummoningOutput.mob('minecraft:wolf')
        .data({ cardinal_components: { "apoli:powers": { Powers: [{ Type: "cot:buff_wolf", Sources: ["cot:test"], Data: {} }] } }, "apathy-spawnType": "spawner" })
    )
    .input('4x minecraft:rotten_flesh')
    .input('4x #farmersdelight:wolf_prey')
    .recipeTime(60) //600
  // .blockBelow('minecraft:hay_block')

})
```
after summoning the wolf just had
`cardinal_components: { "apoli:powers": { Powers: [] } }`
iirc

### Log

https://gist.github.com/800020h/42fed27f3111bf8c44c79b7d792de9d8

### Additional Context

Yes

### Modifications

a lot of mods including origins 1.7.1 which apoli is part of
https://modrinth.com/mod/origins

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by 800020h** [comment]
the spawn type NBT is working as expected

I'm not getting any errors
the summoned wolf doesn't have the power
but the power works if I grant it with commands

the command
`/data merge entity <UUID> { cardinal_components: { "apoli:powers": { Powers: [{ Type: "cot:buff_wolf", Sources: ["cot:test"], Data: {} }] } }, "apathy-spawnType": "spawner" }`
also works 

this also works
```js
BlockEvents.rightClicked('minecraft:glass', event => {

  let buffWolf = event.block.createEntity('minecraft:wolf')
  buffWolf.x+=0.5
  buffWolf.z+=0.5
  buffWolf.mergeFullNBT({ cardinal_components: { "apoli:powers": { Powers: [{ Type: "cot:buff_wolf", Sources: ["cot:test"], Data: {} }] } }, "apathy-spawnType": "spawner" })
  buffWolf.spawn()

})
```
does summoning rituals do something weird?
basically all I have confirmed is the NBT is correct

**Comment by 800020h** [comment]
the power
in `kubejs/data/cot/powers/buff_wolf.json`
```json
{
  "type": "origins:model_color",
  "red": 0.5,
  "green": 0.5,
  "blue": 0.7
}
```

**Comment by 800020h** [comment]
edited to add a log

it should contain the chat outputs from `/data get entity`
both incorrect and correct

mod list has changed a bit since I first had the issue but it hasn't affected it

**Comment by rlnt** [comment]
I'm gonna close this. 1.20 will no longer receive any updates. 1.21 should work perfectly fine with this. The mod has been rewritten from scratch.
